src/js/api/fetchMoviesMain.js

```diff
@@ -0,0 +1,25 @@
+const fetchMoviesMain = async () => {
+    try {
+    const data = await fetch("https://api.themoviedb.org/3/trending/all/day?api_key=4f9b3bc6cd1b3d6e0d830ad9a5ccfefd");
+
+    const response = await data.json();
+    const mostPopularFilms = response.results;
+    
+        return mostPopularFilms
+    } catch (error) {
+        console.log(error.message)
+    }
+
+}
+
+
```

Comment:
tutaj trzeba dodać komentarz że ta część jest do usunięcia - a tak to służy tylko wizualizacji